
---
title: Data Sources
---

# Data Sources Summary
{{% card-container %}}
{{% card title="pfam" %}}
Pfam is a widely used database of protein families, currently containing more than 13,000 manually curated protein families as of release 26.0.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22127870" target="_blank">paper</a>
{{% /card %}}
{{% card title="ensembl-protein" %}}
The UniProt Knowledgebase (UniProtKB) acts as a central hub of protein knowledge by providing a unified view of protein sequence and functional information.
<a href="https://academic.oup.com/database/article/doi/10.1093/database/bar009/463132" target="_blank">paper</a>
{{% /card %}}
{{% card title="mc3" %}}
The Cancer Genome Atlas (TCGA) cancer genomics dataset includes over 10,000 tumor-normal exome pairs across 33 different cancer types, in total >400 TB of raw data files requiring analysis.
<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(18)30096-6" target="_blank">paper</a>
{{% /card %}}
{{% card title="ensembl" %}}
The Ensembl project has been aggregating, processing, integrating and redistributing genomic datasets since the initial releases of the draft human genome, with the aim of accelerating genomics research through rapid open distribution of public data.
<a href="https://academic.oup.com/nar/article/46/D1/D754/4634002" target="_blank">paper</a>
{{% /card %}}
{{% card title="tcga" %}}
 The Cancer Genome Atlas (TCGA) Research Network has profiled and analyzed large numbers of human tumours to discover molecular aberrations at the DNA, RNA, protein, and epigenetic levels. The resulting rich data provide a major opportunity to develop an integrated picture of commonalities, differences, and emergent themes across tumour lineages.
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919969/" target="_blank">paper</a>
{{% /card %}}
{{% card title="ccle" %}}
 (CCLE): a compilation of gene expression, chromosomal copy number and massively parallel sequencing data from 947 human cancer cell lines. When coupled with pharmacological profiles for 24 anticancer drugs across 479 of the cell lines, this collection allowed identification of genetic, lineage, and gene-expression-based predictors of drug sensitivity.
<a href="https://www.sciencedirect.com/science/article/pii/S0092867417306517" target="_blank">paper</a>
{{% /card %}}
{{% card title="myvariant.info" %}}
Efficient tools for data management and integration are essential for many aspects of high-throughput biology. In particular, annotations of genes and human genetic variants are commonly used but highly fragmented across many resources. Here, we describe MyGene.info and MyVariant.info, high-performance web services for querying gene and variant annotation information.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0953-9" target="_blank">paper</a>
{{% /card %}}
{{% card title="gene_enricher" %}}
The BioMart Community Portal (www.biomart.org) is a community-driven effort to provide a unified interface to biomedical databases that are distributed worldwide. The portal provides access to numerous database projects supported by 30 scientific organizations. It includes over 800 different biological datasets spanning genomics, proteomics, model organisms, cancer data, ontology information and more.
<a href="http://nar.oxfordjournals.org/content/43/W1/W589" target="_blank">paper</a>
{{% /card %}}
{{% card title="gdsc" %}}
 The Genomics of Drug Sensitivity in Cancer (GDSC) database (www.cancerRxgene.org) is the largest public resource for information on drug sensitivity in cancer cells and molecular markers of drug response. Data are freely available without restriction. GDSC currently contains drug sensitivity data for almost 75 000 experiments, describing response to 138 anticancer drugs across almost 700 cancer cell lines.
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/" target="_blank">paper</a>
{{% /card %}}
{{% card title="go" %}}
Genomic sequencing has made it clear that a large fraction of the genes specifying the core biological functions are shared by all eukaryotes. Knowledge of the biological role of such shared proteins in one organism can often be transferred to other organisms. The goal of the Gene Ontology Consortium is to produce a dynamic, controlled vocabulary that can be applied to all eukaryotes even as knowledge of gene and protein roles in cells is accumulating and changing.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/10802651" target="_blank">paper</a>
{{% /card %}}
{{% card title="gtex" %}}
Understanding the functional consequences of genetic variation, and how it affects complex human disease and quantitative traits, remains a critical challenge for biomedicine. We present an analysis of RNA sequencing data from 1641 samples across 43 tissues from 175 individuals, generated as part of the pilot phase of the Genotype-Tissue Expression (GTEx) project. We describe the landscape of gene expression across tissues, catalog thousands of tissue-specific and shared regulatory expression quantitative trait loci (eQTL) variants, describe complex network relationships, and identify signals from genome-wide association studies explained by eQTLs.
<a href="http://science.sciencemag.org/content/348/6235/648" target="_blank">paper</a>
{{% /card %}}
{{% card title="g2p" %}}
Precision oncology relies on the accurate discovery and interpretation of genomic variants to enable
individualized therapy selection, diagnosis, or prognosis. However, knowledgebases containing clinical
interpretations of somatic cancer variants are highly disparate in interpretation content, structure, and
supporting primary literature, reducing consistency and impeding consensus when evaluating variants and
their relevance in a clinical setting. With the cooperation of experts of the Global Alliance for Genomics and
Health (GA4GH) and of six prominent cancer variant knowledgebases, we developed a framework for
aggregating and harmonizing variant interpretations to produce a meta-knowledgebase of 12,856 aggregate
interpretations covering 3,437 unique variants in 415 genes, 357 diseases, and 791 drugs.
<a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/11/366856.full.pdf" target="_blank">paper</a>
{{% /card %}}
{{% /card-container %}}
